mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -9,7 +9,8 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-
+	"github.com/hashicorp/terraform-provider-google/version"
```

Comment:
```suggestion
```

These imports (mostly) don't matter, they get added by goimports. Adding a concrete import here locks in a specific library, which is only correct for the GA provider.

Reply:
removed